tests/sentinel/tests/01-conf-update.tcl

```diff
@@ -37,3 +37,11 @@ test "After Sentinel 1 is restarted, its config gets updated" {
 test "New master [join $addr {:}] role matches" {
     assert {[RI $master_id role] eq {master}}
 }
+
+test "Update log level" {
+    set current_loglevel [S 0 SENTINEL CONFIG GET loglevel]
+    assert {[lindex $current_loglevel 1] == {notice}}
```

Comment:
this failed in test-ubuntu-tls: https://github.com/redis/redis/actions/runs/3510377800/jobs/5880112424

because we set loglevel to debug for tls, see #11528